CONTRIBUTING.md

```diff
@@ -69,10 +69,10 @@ Finally, go to GitHub and [make a Pull Request][] :D
 Travis CI and Circle CI will run our test suite. We care about quality, so 
 your PR won't be merged until all tests pass. 
 
-### 8. Stash the changes after review is done
+### 8. Squash the changes after review is done
```

Comment:
This shouldn't be a part of your commit

Reply:
@Swati4star  how do I remove these commits?

Reply:
You have squashed other's commits into your commit.
You can particularly pick your commit, and apply it to PR.
https://stackoverflow.com/questions/16931746/how-to-unsquash-a-commit

Reply:
where can I find the commit reference of just my commit?
should I do git reset --hard HEAD^ and then commit the changes?

Reply:
I don't know the series of command you exactly used to create this commit. 
You can unsquash your commits. Then, you'll get the reference of just your commit.